src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -226,19 +230,11 @@ export const PublishingAccordion = ({
                   : hasClosedTicket
                     ? t('registration.public_page.tasks_panel.files_will_soon_be_rejected')
                     : ''
-                : t('registration.public_page.tasks_panel.registration_will_soon_be_published')}
+                : isWaitingForFileDeletion
```

Comment:
Pass på at dette er godt testet før merge, for jeg antok egentlig at dette ville kreve endringer på linje 224 også. `userCanHandlePublishingRequest` er feks ikke nødvendigvis noen forutsetninger for at dette skal vises lenger. Sjekken for at man kan slette er `userHasAccessRight(registration, 'delete')`, og har ikke noe med publishing request å gjøre, men skal nå også få opp denne bolken. Tror dette først og fremst gjelder for redaktør som er den eneste som har lov til å slette noe som er avpublisert

Reply:
Redaktør har de tilgangene, så det ser ut til at det funker som tiltenkt 🤔

Reply:
ser at `userCanHandlePublishingRequest` også er true om man kan opprette publiseringsforespørsel, så da blir det kanskje riktig pga det ja. Det er en sjekk for `tabErrors` der også som jeg mistenker kan skape noe trøbbel, men har ikke prøvd å trigge noe.
Kommer fort i retur om  det er en annen sjekk som gjør at det ikke vises, som vil være uheldig om det skjer fra mandag, men om du har testet godt går det sikkert fint 👍